examples/hal/Cargo.toml

```diff
@@ -25,7 +25,9 @@ env_logger = "0.5"
 image = "0.18"
 log = "0.4"
 winit = "0.13"
+glsl-to-spirv = "0.1.4"
 gfx-hal = { path = "../../src/hal", version = "0.1" }
+gfx-backend-empty = { path = "../../src/backend/empty", version = "0.1" }
```

Comment:
where is it used?

Reply:
No where currently, used to be used. Removed.